### PR TITLE
Minor documentation updates

### PR DESCRIPTION
**Pull request by zakaluka**

### CODE REVIEW

**Comment by zakaluka** [comment]
The error appears to be with the `just-extend` package.

**Comment by zakaluka** [comment]
The Travis CI server is **VERY** temperamental...

File: src/adapters/IDB.js [hunk 1]
```diff
@@ -189,7 +189,7 @@ export default class IDB extends BaseAdapter {
   }
 
   /**
-   * Returns a transaction and a store objects for this collection.
+   * Returns a transaction and a object store for this collection.
```
Comment (glasserc) [inline src/adapters/IDB.js]: This should be `an object store`.

Comment (zakaluka) [reply src/adapters/IDB.js]: Updated.

**Pull request merged**